cypress/plugins/index.js

```diff
@@ -82,7 +83,19 @@ module.exports = (on, config) => {
   on('task', {
     appendFile: async ({ filename, data }) => {
       try {
-        fs.appendFileSync(filename, data)
+        await fsp.appendFile(filename, data)
+        return Promise.resolve(null)
```

Comment:
I would write this with a `.then` e.g.

`await fsp.appendFile(filename, data).then(() => null) // Cypress won't accept an undefined result`

or with my preferred alternative to comments:

```
const makeSureCypressCanInterpretTheResult = result => result === undefined ? null : result

await fsp.appendFile(filename, data).then(makeSureCypressCanInterpretTheResult)
```

Reply:
Changed as suggested